config/detekt/detekt-config.yml

```diff
@@ -555,7 +555,7 @@ style:
     ignoreNumbers: ['-1', '0', '1', '2']
     ignoreHashCodeFunction: true
     ignorePropertyDeclaration: false
-    ignoreLocalVariableDeclaration: false
+    ignoreLocalVariableDeclaration: true
```

Comment:
Not sure if this is a good idea. I think it's better to suppress it on the classes themselves, if you don't feel like fixing the warnings now

Reply:
It's not really about that. How would you define a number that has a specific meaning only for that method? 
Do you think it should also be declared in the companion object?
I just thought declaring a local variable should be enough to give the number meaning.

Reply:
If you want to go full best practices then declaring it in a companion object is the way. But I do agree that it feels a bit tedious if the "magic number" is only used in one place. This check is useful in general I find. And for places where the magic number is used once, the meaning is clear and you're pretty sure you won't use the same number any time soon I would suppress the warning